HOME: /t3home/frejalom
USER: frejalom
SLURM_JOB_ID: 244134
HOSTNAME: t3wn61
CMSSW RELEASE BASE  /cvmfs/cms.cern.ch/slc7_amd64_gcc12/cms/cmssw/CMSSW_14_2_0_pre4
%MSG-i AlpakaService:  (NoModuleName) 02-Apr-2025 08:41:50 CEST pre-events
AlpakaServiceSerialSync succesfully initialised.
Found 1 device:
  - Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz
%MSG
02-Apr-2025 08:41:55 CEST  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root
[2025-04-02 08:41:56.848639 +0200][Error  ][XRootDTransport   ][215182] [storage01.lcg.cscs.ch:1095.0] Authentication with gsi failed: login failed
[2025-04-02 08:41:56.848695 +0200][Error  ][XRootDTransport   ][215182] [storage01.lcg.cscs.ch:1095.0] No protocols left to try
[2025-04-02 08:41:56.848712 +0200][Error  ][AsyncSock         ][215182] [storage01.lcg.cscs.ch:1095.0] Socket error while handshaking: [FATAL] Auth failed
[2025-04-02 08:41:56.848802 +0200][Error  ][PostMaster        ][215182] [storage01.lcg.cscs.ch:1095] elapsed = 0, pConnectionWindow = 31 seconds.
[2025-04-02 08:41:56.848812 +0200][Error  ][PostMaster        ][215182] [storage01.lcg.cscs.ch:1095] Unable to recover: [FATAL] Auth failed.
[2025-04-02 08:41:56.848822 +0200][Error  ][XRootD            ][215182] [storage01.lcg.cscs.ch:1095] Impossible to send message kXR_open (file: /store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root, mode: 0660, flags: kXR_open_read kXR_async kXR_retstat ). Trying to recover.
[2025-04-02 08:44:52.564314 +0200][Error  ][XRootD            ][215182] [cms-xrd-global.cern.ch:1094] Unable to get the response to request kXR_open (file: /store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-int.cr.cnaf.infn.it,cms03.lcg.cscs.ch&triedrc=srverr, mode: 0460, flags: kXR_open_read kXR_async kXR_retstat )
%MSG-w XrdAdaptorInternal:  file_open 02-Apr-2025 08:44:52 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-int.cr.cnaf.infn.it,cms03.lcg.cscs.ch&xrdcl.requuid=5ebcbb92-8050-4fbd-96ce-ad7b6dedf5f8.
%MSG
[2025-04-02 08:44:52.683927 +0200][Error  ][XRootDTransport   ][215182] [storage01.lcg.cscs.ch:1095.0] Authentication with gsi failed: login failed
[2025-04-02 08:44:52.683998 +0200][Error  ][XRootDTransport   ][215182] [storage01.lcg.cscs.ch:1095.0] No protocols left to try
[2025-04-02 08:44:52.684015 +0200][Error  ][AsyncSock         ][215182] [storage01.lcg.cscs.ch:1095.0] Socket error while handshaking: [FATAL] Auth failed
[2025-04-02 08:44:52.684097 +0200][Error  ][PostMaster        ][215182] [storage01.lcg.cscs.ch:1095] elapsed = 0, pConnectionWindow = 31 seconds.
[2025-04-02 08:44:52.684109 +0200][Error  ][PostMaster        ][215182] [storage01.lcg.cscs.ch:1095] Unable to recover: [FATAL] Auth failed.
[2025-04-02 08:44:52.684119 +0200][Error  ][XRootD            ][215182] [storage01.lcg.cscs.ch:1095] Impossible to send message kXR_open (file: /store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=, mode: 0660, flags: kXR_open_read kXR_async kXR_retstat ). Trying to recover.
[2025-04-02 08:47:47.594252 +0200][Error  ][XRootD            ][215182] [cms-xrd-global.cern.ch:1094] Unable to get the response to request kXR_open (file: /store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-int.cr.cnaf.infn.it,cms03.lcg.cscs.ch&triedrc=srverr, mode: 0460, flags: kXR_open_read kXR_async kXR_retstat )
%MSG-w XrdAdaptorInternal:  file_open 02-Apr-2025 08:47:47 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-int.cr.cnaf.infn.it,cms03.lcg.cscs.ch&xrdcl.requuid=860b9d73-32e6-475d-96f5-d9f506f5f35c.
%MSG
----- Begin Fatal Exception 02-Apr-2025 08:47:47 CEST-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing input source of type PoolSource
   [2] Calling RootInputFileSequence::initTheFile()
   [3] Calling StorageFactory::open()
   [4] Calling XrdFile::open()
Exception Message:
Failed to open the file 'root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root'
   Additional Info:
      [a] Input file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root could not be opened.
      [b] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.
      [c] Last URL tried: root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/e36b3f7f-5915-40dc-9f39-637a0259e803.root?tried=+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-int.cr.cnaf.infn.it,cms03.lcg.cscs.ch&xrdcl.requuid=860b9d73-32e6-475d-96f5-d9f506f5f35c
      [d] Problematic data server: cms-xrd-global.cern.ch:1094
      [e] Disabled source: cms-xrd-global.cern.ch:1094
----- End Fatal Exception -------------------------------------------------
logfile
pv_cfg.py
cp: cannot stat 'pv.root': No such file or directory
